### PR TITLE
Add dahboards, saved searches and macros

### PR DESCRIPTION
**Pull request by puja-sacumen**

### CODE REVIEW

File: Eclecticiq-ic-app/default/data/ui/alerts/create_eclecticiq_sighting.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
+<form class="form-horizontal form-complex">
```
Comment (pranav-sacumen) [inline Eclecticiq-ic-app/default/data/ui/alerts/create_eclecticiq_sighting.html]: This file should be in TA. Remove it from here.

File: Eclecticiq-ic-app/default/data/ui/nav/default.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  <collection label="Dashboards">
+        <a href="eiq_matches_by_ip">Matched IP's</a>
+        <a href="eiq_matches_by_domain">Matched Domains and URLs</a>
+        <a href="eiq_matches_by_filehashes">Matched File hashes</a>
+   		<a href="eiq_matches_by_email">Matched Emails</a>
+    <a href="eiq_matches">All Matches</a>
+  </collection>
+  <collection label="Information">
+    <a href="eiq_db_info">Observables DB Info</a>
+    <a href="setup_view_dashboard">Edit app settings</a>
```
Comment (pranav-sacumen) [inline Eclecticiq-ic-app/default/data/ui/nav/default.xml]: setup view is not required.

File: Eclecticiq-ic-app/default/data/ui/views/setup_view_dashboard.xml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<!--
```
Comment (pranav-sacumen) [inline Eclecticiq-ic-app/default/data/ui/views/setup_view_dashboard.xml]: This file is not not required. Remove it.

File: Eclecticiq-ic-app/metadata/local.meta [hunk 4]
```diff
@@ -0,0 +1,59 @@
+[views/home]
```
Comment (pranav-sacumen) [inline Eclecticiq-ic-app/metadata/local.meta]: There should be no local.meta. Remove it.

**Review by pranav-sacumen — CHANGES_REQUESTED** [review]

File: Eclecticiq-ic-app/default/data/ui/alerts/create_eclecticiq_sighting.html (hunk 1 above)
Comment (puja-sacumen) [reply Eclecticiq-ic-app/default/data/ui/alerts/create_eclecticiq_sighting.html]: Okay

File: Eclecticiq-ic-app/default/data/ui/nav/default.xml (hunk 2 above)
Comment (puja-sacumen) [reply Eclecticiq-ic-app/default/data/ui/nav/default.xml]: Okay

File: Eclecticiq-ic-app/default/data/ui/views/setup_view_dashboard.xml (hunk 3 above)
Comment (puja-sacumen) [reply Eclecticiq-ic-app/default/data/ui/views/setup_view_dashboard.xml]: Okay

File: Eclecticiq-ic-app/metadata/local.meta (hunk 4 above)
Comment (puja-sacumen) [reply Eclecticiq-ic-app/metadata/local.meta]: Okay

**Review by pranav-sacumen — APPROVED** [review]

**Review by pranav-sacumen — APPROVED** [review]

**Pull request merged**